libraries/adaptive-expressions/src/functionUtils.internal.ts

```diff
@@ -223,12 +231,14 @@ export class InternalFunctionUtils {
     }
 
     /**
-     * Get the value of a path from a memory
+     * Get the value of a path from a memory.
+     *
      * @param state Memory.
      * @param path Path string.
      * @param options Options.
+     * @returns The value of a path from a memory.
      */
-    public static wrapGetValue(state: MemoryInterface, path: string, options: Options): any {
+    public static wrapGetValue(state: MemoryInterface, path: string, options: Options): unknown {
```

Comment:
We should leave it as `any` for now. no?

Reply:
It was an auto-fix I think. I'll change it back.